### DISCUSSION TITLE
Any estimate on when the new version will be available?

### DISCUSSION THREAD

**Discussion by vfonic** [Shopify/dawn#2217] [General]
Hi, thank you for all of your work! I just started using Dawn for a new clean theme refresh for a client and I'm really happy about it!

I'm trying to figure out a best way to make theme customizations and to stay up to date with the latest changes here (`upstream').

I figured perhaps the best would be to merge-in any changes when the new version is released (so that I catch the most stable changes, not in the middle of development, not the bleeding-edge `main` branch).

With that in mind, I checked out v7.0.1 tag and started making customizations from there.
However, I noticed that _after_ v7.0.1 was released all files were Prettier formatted. This means that there are many files that were changed after v7.0.1. I'm stuck considering just checking out the `main` branch instead.

I noticed that the new version hasn't been released since September. Any chance that a new version is released so that I'm saved from many merge conflicts?
https://github.com/Shopify/dawn/search?q=prettier&type=commits

Thanks!

Related discussion: https://github.com/Shopify/dawn/discussions/1863

**Comment by nelsojona** [comment]
Also, calling out that the current build is failing.

<img width="1200" alt="Screen Shot 2023-01-23 at 4 54 10 PM" src="https://user-images.githubusercontent.com/688996/214158573-04ab776c-c5de-4923-9c1a-3df46a86bb11.png">

**Comment by vfonic** [comment]
Looks like Shopify Dawn team released v8.0.0 yesterday! 🎉

I don't know how to close this discussion.

The build is currently still failing, but the main issue has already been fixed (unknown Liquid filter). Now the build failure is due to Lighthouse score being too low (by 0.01 point). So this is more of an intermittent build failure.

I assume the best fix would be to lower the required Lighthouse page speed score so that the build always passes.
